tests/test_description.py

```diff
@@ -34,5 +34,13 @@ def test_lambda_content_description_on_timeout():
     assert '3 == 4' == e.value.description
 
 
+def test_lambda_content_with_captures():
+    c1 = 321
```

Comment:
These (c1 & c2) are redundant, since their values have no meaning, you can inline the values